### DISCUSSION TITLE
A new node with list

### DISCUSSION THREAD

**Discussion by okolenmi** [comfyanonymous/ComfyUI#1430] [Ideas]
Custom nodes is good, but it will be better to make some more flexible and advanced nodes, at least as a sample. I would like to see a node with expandable+scrollable list, where you can add new items, enable or disable them, and then generate a result based on items processing.
For example, it will be nice to see thing like this for embeddings (I'm didn't think a lot about this design):
![изображение](https://github.com/comfyanonymous/ComfyUI/assets/55319814/d8b0bb55-f512-4859-8133-073a807646f6)

This is much more comfortable than [find embeddings folder] -> [copy a name of embedding] -> [paste the name in window] -> [add word "embedding"] -> [add brackets and ":1.2"] -> **let's do it again for extra one**

**Comment by cdsama** [comment]
try https://github.com/pythongosssss/ComfyUI-Custom-Scripts, it helps a lot
